.github/workflows/main_workflow.yml

```diff
@@ -155,7 +155,6 @@ jobs:
     continue-on-error: True
     needs:
       - lock_check
-      - copyright_doc_and_pipfile_check
```

Comment:
removing this temporarily in order to see the results

Reply:
You may cherry pick 218f1ebd4898e774611534702c0c5595f76affdc instead.

Reply:
This is now resolved by cherry picking. Thanks @Adamantios . 
Force pushed to not pollute the repo with unnecessary commits and their reverts.